README.md

```diff
@@ -1,76 +1,221 @@
 
-```bash
-npm install
+- run command
+
+```
+npm run build
 ```
 
-### Usage
+## Start server
 
-```bash
+```
 npm start
 ```
 
-### Build
+### Prerequisites <a name="prerequisites"></a>
 
-```bash
-npm run build
+In order to run this project you need:
+
+- [x] Basic Knowledge of **HTML**, **CSS**, & **JavaScript**.
+- [x] Basic knowledge of **ES6** syntax.
+- [x] Basic knowledge of **Git**, **GitHub** &**Gitflow**.
+- [x] Basic knowledge of **Linters**.
+- [x] Basic knowledge of **Webpack**.
+- [x] Basic knowledge of **VS Code** or any other code-editors.
+
+### Setup
+
+Clone this repository to your desired folder:
+
+    _cd PATH_
+
+    _git clone  https://github.com/The-Normal-One1/math-magicians.git
+
+_cd Javascript-Group-Capstone_
+
+_npm install_
+
+_npm run build_
+
+_npm start_
+
+### Install
+
+Install this project with:
+
+```sh
+  npm install
+```
+
+### Usage
+
+To run the project, execute the following command:
+
+```sh
+  npm start
 ```
 
 ### Run tests
 
-```bash
-npm test
+To run tests, run the following command:
+
+```sh
+  npm test
 ```
 
-## Authors
+### Deployment <a name="triangular_flag_on_post-deployment"></a>
+
+You can deploy this project using:
+
+```sh
+  npm start
+```
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+<!-- AUTHORS -->
+
+## 👥 Authors <a name="authors"></a>
+
+👤 **Amk Lass**
 
-**Amk lass**
 
 GitHub:[@amklass11](https://github.com/amklass11)
+
 LinkedIn:[@AmkLAss](https://www.linkedin.com/in/amk-lass-521565196/)
 
-## 🤝 Contributing
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+## 🔭 Future Features <a name="future-features"></a>
+
+- [ ] **Implement additional pages**
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+## 🤝 Contributing <a name="contributing"></a>
 
 Contributions, issues, and feature requests are welcome!
 
-Feel freedto check the [issues page](../../issues/).
+Feel free to check the [issues page](https://github.com/The-Normal-One1/math-magicians/issues).
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
 
-## Show your support
+<!-- SUPPORT -->
+
+## ⭐️ Show your support <a name="support"></a>
 
 Give a ⭐️ if you like this project!
 
-## Acknowledgments
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+## 🙏 Acknowledgments <a name="acknowledgements"></a>
+
+- Thanks to [Microverse](https://www.microverse.org/) for giving this opportunity to learn ...
+- Code Reviewers & Coding Partners.
+- Hat tip to anyone whose code was used.
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+<!-- FAQ (optional) -->
+
+## ❓ FAQ <a name="faq"></a>
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+<!-- LICENSE -->
 
-- Microverse
-- App Logo from [FlatIcons](https://www.flaticon.com/free-icon/planet_3212567?term=space&page=1&position=19&page=1&position=19&related_id=3212567&origin=style)
+## 📝 License <a name="license"></a>
 
-## 📝 License
+This project is [MIT](./LICENSE) licensed.
```

Comment:
[OPTIONAL] This link opens a 404 page. KIndly rename  this link to `[MIT](./MIT.md)` since your licence is already renamed to `MIT.md`